admin/Model/Config/Source/Mode.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Pronko\LiqPayAdmin\Model\Config\Source;
+
+use Magento\Framework\Option\ArrayInterface;
+use Pronko\LiqPayAdmin\Model\ModeStatusInterface;
+
+class Mode implements ArrayInterface
+{
+
+    /**
+     * Return array of options as value-label pairs
+     *
+     * @return array Format: array(array('value' => '<value>', 'label' => '<label>'), ...)
+     */
+    public function toOptionArray()
+    {
+        return [
+            ['value' => ModeStatusInterface::PRODUCTION, 'label' => __('Production')],
+            ['value' => ModeStatusInterface::SANDBOX, 'label' => __('Sandbox')],
+        ];
+    }
+}
```

Comment:
Please add an empty line.

Reply:
@mcspronko maybe labels better move into the  ModeStatusInterface  also ?

Reply:
It is not a good idea as a label isn't really a part of an API or interface.

Reply:
@mcspronko ok, i understand